cubi-tools/update_workflow.py

```diff
@@ -33,6 +43,17 @@ def main():
         "CITATION.md",
         "LICENSE",
         ".editorconfig",
+        "init.py",
```

Comment:
here is a logical problem - this script should update the workflow template, which includes the metadata, yes, but the update of the metadata is realized in a different script. What this results in is one big pile of duplicated code. So focus here first only on the template update and leave the metadata update in the domain of the other script.

Reply:
A switch has been implemented so the user can select to also update the metadata; however, the default is set to False